%YAML:1.0

R_imu_cam: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data: [0.010309024186486937, -0.9999260102291188,-0.006457405639991585,
           0.9998300378494687, 0.010208895415655062,  0.015351673144095974, 
           -0.01528461429848226, -0.006614568895187682, 0.9998613044038049]

t_imu_cam: !!opencv-matrix
  rows: 3
  cols: 1
  dt: d
  data: [-0.0019196,-0.0392691, 0.0072418]

intrinsics: !!opencv-matrix
  rows: 4
  cols: 1
  dt: d
  data: [377.37396383958304,377.33612468606094,385.5019347556897,231.1887530656648]

#distortion_coeffs: !!opencv-matrix
#  rows: 4
#  cols: 1
#  dt: d
#  data: [-0.31460157366941205,0.09123078956376497,-0.0009452600710251895,-0.0018935763084249112]

distortion_coeffs: !!opencv-matrix
  rows: 4
  cols: 1
  dt: d
  data: [0.,0.,0.,0.]